WeatherApp/Services/WeatherApiService.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  WeatherApiService.swift
+//  WeatherApp
+//
+//  Created by Lucija Balja on 03/08/2020.
+//  Copyright © 2020 Lucija Balja. All rights reserved.
+//
+
+import Foundation
+
+class WeatherApiService {
+    var apiKey = "56151fef235e6cebb33750525932d021"
```

Comment:
Both of these fields can be `private let`

Reply:
@istaudacher I get a "Cannot use instance member 'apiKey' within property initializer; property initializers run before 'self' is available" error on apiKey when both are constants.